src/jace/translator/translated_jaxpr_sdfg.py

```diff
@@ -0,0 +1,85 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
+#
+# Copyright (c) 2024, ETH Zurich
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+import dace
+
+from jace import util
+
+
+class TranslatedJaxprSDFG:
```

Comment:
Why not just `JaxprSDFG`? The translated part seems a bit redundant to me...

Reply:
I decided to keep it that way, mostly because in docstrings I refered to it as "translated SDFG" it is nice to refer to it.